src/dotnet/commands/dotnet-msbuild/MSBuildLogger.cs

```diff
@@ -66,16 +69,43 @@ internal static void FormatAndSend(ITelemetry telemetry, TelemetryEventArgs args
             if (args.EventName == TargetFrameworkTelemetryEventName)
             {
                 var newEventName = $"msbuild/{TargetFrameworkTelemetryEventName}";
-                Dictionary<string, string>  maskedProperties = new Dictionary<string, string>();
-                if (args.Properties.TryGetValue(TargetFrameworkVersionTelemetryPropertyKey, out string value))
+                Dictionary<string, string> maskedProperties = new Dictionary<string, string>();
+                if (args.Properties.TryGetValue(TargetFrameworkVersionTelemetryPropertyKey, out string targetFrameworkVersionValue))
+                {
+                    maskedProperties.Add(TargetFrameworkVersionTelemetryPropertyKey, Sha256Hasher.HashWithNormalizedCasing(targetFrameworkVersionValue));
+                }
+
+                if (args.Properties.TryGetValue(UseWindowsFormsTelemetryPropertyKey, out string hasDesktopUIReferenceValue))
                 {
-                    maskedProperties.Add(TargetFrameworkVersionTelemetryPropertyKey, Sha256Hasher.HashWithNormalizedCasing(value));
+                    maskedProperties.Add(UseWindowsFormsTelemetryPropertyKey, SanitizeToOnlyTrueFalseEmpty(hasDesktopUIReferenceValue));
+                }
+
+                if (args.Properties.TryGetValue(UseWPFTelemetryPropertyKey, out string hasAnyPageValue))
```

Comment:
Rename `hasAnyPageValue` to `useWPFValue`.

Reply:
fixed